streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/system/mapper/RoleMenuMapper.java

```diff
@@ -20,5 +20,7 @@
 import org.apache.streampark.console.system.entity.RoleMenu;
 
 import com.baomidou.mybatisplus.core.mapper.BaseMapper;
+import org.springframework.stereotype.Component;
 
+@Component
```

Comment:
Why add this annotation?

Reply:
I add it to solve  error-red highlight lines caused by error cache of my IDE.

And I removed it, now.
Thank you.